docs/notebooks/formulary/coulomb.ipynb

```diff
@@ -0,0 +1,5759 @@
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "bmin = 1.05e-11 m\n",
+      "bmax = 2.18e-03 m\n"
+     ]
+    }
+   ],
+   "source": [
+    "solar_corona = {\n",
+    "    \"T\": 1e6 * u.K,\n",
+    "    \"n_e\": 1e15 * u.m ** -3,\n",
+    "    \"species\": [\"e-\", \"p+\"],\n",
+    "}\n",
+    "\n",
+    "bmin, bmax = impact_parameter(**solar_corona)\n",
+    "\n",
+    "print(f\"{bmin = :.2e}\")\n",
+    "print(f\"{bmax = :.2e}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "428e109d",
+   "metadata": {},
+   "source": [
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "53ba7ecf",
+   "metadata": {},
+   "source": [
+    "When we can calculate the Coulomb logarithm, we find that it is ∼20 (a common value for astrophysical plasma)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "7077f58d",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "19.150697742645406"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "Coulomb_logarithm(**solar_corona)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "036e5a64",
+   "metadata": {},
+   "source": [
+    "Our next goal is to visualize the Coulomb logarithm.  Let's start by creating a function to plot $\\ln{Λ}$ and $b$.  To get a \"Λ\" in a Jupyter notebook, type `\\Lambda` and press tab."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "f3863ccf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def plot_coulomb(T, n_e, species, **kwargs):\n",
+    "    \"\"\"Plot the Coulomb logarithm.\"\"\"\n",
+    "\n",
+    "    bmin, bmax = impact_parameter(T, n_e, species, **kwargs)\n",
+    "    ln_Λ = Coulomb_logarithm(T, n_e, species, **kwargs)\n",
+    "\n",
+    "    with quantity_support():\n",
+    "\n",
+    "        fig, axes = plt.subplots(nrows=1, ncols=2)\n",
+    "\n",
+    "        fig.set_size_inches(10, 4)\n",
+    "\n",
+    "        axes[0].semilogx(T, ln_Λ)\n",
+    "        axes[0].set_xlabel(\"T (K)\")\n",
+    "        axes[0].set_ylabel(\"$\\ln{Λ}$\")\n",
+    "        axes[0].set_title(\"Coulomb logarithm\")\n",
+    "\n",
+    "        axes[1].loglog(T, bmin, label=\"$B_\\mathrm{min}$\")\n",
+    "        axes[1].loglog(T, bmax, label=\"$B_\\mathrm{min}$\")\n",
+    "        axes[1].set_xlabel(\"T (K)\")\n",
+    "        axes[1].set_ylabel(\"b (m)\")\n",
+    "        axes[1].set_title(\"Impact parameters\")\n",
+    "        axes[1].legend()\n",
+    "\n",
+    "        fig.tight_layout()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f0740c74",
+   "metadata": {},
+   "source": [
+    "Let's calculate the Coulomb logarithm and minimum/maximum impact parameters over a range of temperatures in a high density plasma."
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
What is happening in this cell?  Doesn't look like the code I expect...

Reply:
Changed "calculate" to "plot"